tests/test_parser.py

```diff
@@ -0,0 +1,105 @@
+import unittest
+import warnings
+from pathlib import Path
+
+import rdflib
+
+_TEST_DIR = Path(__file__).resolve().parent
+
+
+class TestGraphParsing(unittest.TestCase):
+    def test_parsing_ox_turtle_bulk_load(self):
```

Comment:
tests allow for paralization, would be good to switch from unittest to pytest in coming PRs

Reply:
Yes!